src/selectedtests/app/controllers/project_task_mappings_controller.py

```diff
@@ -65,7 +67,7 @@ class TaskMappings(Resource):
         @ns.response(400, "Bad request")
         @ns.response(404, "Evergreen project not found")
         @ns.expect(parser)
-        def get(self, project: str):
+        def get(self, project: str) -> Response:  # type: ignore
```

Comment:
Why all the `type: ignore`s?

Reply:
mypy doesn't like that we don't return anything (when calling `abort`).